speech/docs/index.rst

```diff
@@ -19,16 +19,13 @@ See: `Speech Asynchronous Recognize`_
 
     >>> from google.cloud import speech
     >>> client = speech.SpeechClient()
-    >>> operation = client.long_running_recognize(
-    ...     audio=speech.types.RecognitionAudio(
-    ...         uri='gs://my-bucket/recording.flac',
-    ...     ),
-    ...     config=speech.types.RecognitionConfig(
-    ...         encoding='LINEAR16',
-    ...         language_code='en-US',
-    ...         sample_rate_hertz=44100,
-    ...     ),
-    ... )
+    >>> audio = speech.types.RecognitionAudio(
+    ...     uri='gs://my-bucket/recording.flac')
+    >>> config=speech.types.RecognitionConfig(
```

Comment:
For assignments, please have spaces on either side of the `=` (versus when passing as arguments).  Likewise below, three times.